app/views/home/me.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="inset">
+ <h2>Welcome, <%= current_user.email_address %></h2>
+</div>
```

Comment:
Ah. In that case, game to change the logic in the header to make everything go through `current_user`?